spec/workers/polling_alert_check_worker_spec.rb

```diff
@@ -0,0 +1,88 @@
+RSpec.describe PollingAlertCheckWorker do
+  include SearchAlertListHelpers
+  include NotifyRequestHelpers
+
+  describe "#perform", caching: true do
+    def perform
+      described_class.new.perform("medical_safety_alert")
+    end
+
+    context "there are no alerts older than an hour" do
+      before { stub_medical_safety_alert_feed(content_id: SecureRandom.uuid, age: 30.minutes) }
```

Comment:
Could rename `stub_medical_safety_alert_feed` to `stub_medical_safety_alert_query` as feed reminds me of atom 😄